tripper/triplestore.py

```diff
@@ -160,6 +161,26 @@ def __init__(
             kwargs: Keyword arguments passed to the backend's __init__()
                 method.
 
+        Attributes:
+            backend_name: Name of backend.
+            base_iri: Assigned to the `base_iri` argument.
+            closed: Whether the triplestore is closed.
+            kwargs: Dict with additional keyword arguments.
```

Comment:
kwargs should be at the end.

Reply:
`kwargs` is an attribute assigned to the `kwargs` argument. Hence `ts.kwargs` is a dict with additional keyword arguments.